pandera/typing/dask.py

```diff
@@ -32,26 +31,6 @@ class DataFrame(DataFrameBase, dd.DataFrame, Generic[T]):
         *new in 0.8.0*
         """
 
-        def __setattr__(self, name: str, value: Any) -> None:
```

Comment:
Was this intended to be removed?

Reply:
yep, this isn't necessary since `DataFrameBase` implements the same method